src/main/java/duke/TaskList.java

```diff
@@ -0,0 +1,78 @@
+package duke;
+
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+import java.util.ListIterator;
+
+import duke.models.Task;
+
+
+
+/**
+ * Contains the task list and has operations to add/delete tasks in the list
+ */
+public class TaskList {
+    private List<Task> tasks;
```

Comment:
I like that you named the List<Task> object as tasks, with the plural 's'. Good coding standard